src/components/common/ColonyActions/ActionDetailsPage/DefaultMotion/MotionCountdown/MotionCountdown.tsx

```diff
@@ -43,37 +44,40 @@ const displayName = 'common.ColonyActions.ActionDetailsPage.MotionCountdown';
 
 interface MotionCountdownProps {
   motionState: MotionState;
+  refetchMotionState: RefetchMotionState;
   motionData: MotionData;
 }
 
-const MotionCountdown = ({ motionState, motionData }: MotionCountdownProps) => {
+const MotionCountdown = ({
+  motionState,
+  motionData,
+  refetchMotionState,
+}: MotionCountdownProps) => {
   // const motionDomain = 1;
   const motionId = 1;
```

Comment:
This needs to come from the motion data...

Reply:
Yup, my bad. I missed this part when I was updating the component